manager-dashboard/app/utils/common.tsx

```diff
@@ -308,3 +308,14 @@ export const iconMap = listToMap(
     (icon) => icon.key,
     (icon) => icon.component,
 );
+
+export function formatProjectTopic(str?: string) {
+    if (isNotDefined(str)) return null;
+
+    // Note: this will remove start and end space
+    const removeFirstAndLastSpace = str.trim();
```

Comment:
The variable name should be different.
We should also add a NOTE that this validation mirrors feature from the firebase function